prac_05/emails.py

```diff
@@ -0,0 +1,41 @@
+"""
+CP1404/CP5632 Practical
+Get and store name and email
+"""
+
+
+def main():
+    """
+    Gets user input of names and emails
```

Comment:
Make sure to write your docstrings in the imperative voice.

So for "gets user input of names and emails" just change it to "get user input of names and emails".

see week 3 - functions slides for more details